products/nativescript/tns.py

```diff
@@ -221,6 +221,30 @@ def plugin_remove(plugin_name, path=None, log_trace=False, verify=True):
             assert "Successfully removed plugin {0}".format(plugin_name.replace("@", "")) in result.output
         return result
 
+    @staticmethod
+    def plugin_create(plugin_name, path=None, log_trace=False, verify=True):
+        # Execute plugin create command
+        result = Tns.exec_command(command="plugin create --includeTypeScriptDemo=y " + plugin_name, path=path,
```

Comment:
Do we have a case without TypeScript demo?
Does it make sense to pass it as param of `plugin_create` method?